scripts/urlgrabber-ext-down

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/python
+#! /usr/bin/python3
```

Comment:
Strictly speaking, this should be left alone for now...

Reply:
Mind elaborating? :)

Reply:
People can still install this with Python 2, and setuptools rewrites the shebang when you install it. :)

Reply:
> setuptools rewrites the shebang when you install it. :)

Oh, I didn't know that!  I'm going to drop that commit then, thanks :)